src/EditModel/EditModelForm.component.js

```diff
@@ -218,14 +221,33 @@ export default React.createClass({
         return this.renderForm();
     },
 
+    /*
+        Will render the fields that are relevant for the current "step"
+        This is done by setting the fields "component" to an empty function
+        if it is not relevant. The component is saved in the variable "hiddenComponent".
+        When the stepper is set to a step in which the field is supposed to be rendered
+        the hiddenComponent will be set back to the component variable.
+
+        **HACK**
+        Since the field rules in EditModel/form-rules/index.js uses the same way to hide and show
+        fields with the HIDE_FIELD and SHOW_FIELD rule, this causes the rule not work using 
+        the stepper. Setting the style to 'display: none' will would work for the stepper and 
+        still allowing the rules to use "hiddenComponent" as a way to show and hide fields, 
+        but since not all component does not spread and merge style, it will not always work.
+        Whe all components uses spread style this hack can be removed.
```

Comment:
Did you investigate how many components there are which need to use the spread style hack?